examples/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -68,47 +68,50 @@ const createFishjamRoom = async () => {
     console.log(e);
   }
 };
-const addPeerToRoom = async (
+const addParticipiantToRoom = async (
   roomId: string,
   enableSimulcast: boolean = true,
 ) => {
-  const { addPeer } = RoomApiFp(config);
-  const addPeerRequest: AddPeerRequest = {
+  const { addPeer: addParticipiant } = RoomApiFp(config);
```

Comment:
Do we want to rename `@fishjam-cloud/fishjam-openapi` functions to `Participants`?

Reply:
maybe someday?  there are few other tickets for renames